en/debug/system_console.md

```diff
@@ -70,6 +82,13 @@ The complete wiring is shown below.
 
 ![Console Debug](../../assets/console/console_debug.jpg)
 
+
+### Snapdragon Flight
```

Comment:
@bkueng Not sure about this section, and how it works with the other snapdragon flight section. Again though, I  proposed to move this into the Snapdragon docs.

What I THINK is that essentially this is essentially a debug adaptor board to which you connect the FTDI cable?

Reply:
I think so, but I'm not entirely familiar either

Reply:
I've moved them. There were stub docs that the content just slipped into.